src/assets/index.js

```diff
@@ -6,12 +6,16 @@ const show_landing_container = () => {
     landingContainer.classList.remove("hidden");
     inputContainer.classList.add("hidden");
     resultContainer.classList.add("hidden");
-} 
+}
 
 const show_input_container = () => {
     landingContainer.classList.add("hidden");
     inputContainer.classList.remove("hidden");
     resultContainer.classList.add("hidden");
+    const showImage = document.getElementById("showImage");
+    showImage.src = "";
```

Comment:
great work!!
 also clear input-field, currently only image is deleted but input is set to the old file